pipeline.sh

```diff
@@ -56,11 +60,13 @@ examples:
   pipeline.sh -i in.vcf.gz -o out.vcf.gz -t sample0/HP:0000123
   pipeline.sh -i in.vcf.gz -o out.vcf.gz -t sample0/HP:0000123,sample1/HP:0000234
   pipeline.sh -i in.vcf.gz -o out.vcf.gz --ann_vep "--refseq --exclude_predicted --use_given_ref"
+  pipeline.sh -i in.vcf.gz -o out.vcf.gz --args_preprocess "--filter_read_depth -1"
```

Comment:
Escape double-quotes because the echo value is already in double-quotes. (existing bug in --ann_vep as well?)